ext/strings.go

```diff
@@ -44,6 +60,40 @@ import (
 //	'hello'.charAt(5)  // return ''
 //	'hello'.charAt(-1) // error
 //
+// # Format
+//
+// Returns a new string with substitutions being performed, printf-style.
+// The valid formatting clauses are:
+//
+// `%s` - substitutes a string.
+// `%d` - substitutes an integer.
+// `%f` - substitutes a double with fixed-point precision. The default precision is 6, but
+// this can be adjusted. The strings `Infinity`, `-Infinity`, and `NaN` are also valid input
+// for this clause.
+// `%e` - substitutes a double in scientific notation. The default precision is 6, but this
+// can be adjusted.
+// `%E` - same as a above, but with an uppercase E.
+// `%b` - substitutes an integer with its equivalent binary string.
+// `%h` - substitutes an integer with its equivalent in hexadecimal, or if given a string, will
+// output each character's equivalent in hexadecimal.
+// `%H` - same as above, but with A-F capitalized.
+// `%o` - substitutes an integer with its equivalent in octal.
```

Comment:
Also document %h, %H for bytes and CEL date, time and duration support?

Reply:
Right, I've added documentation for both - fixed

Reply:
The standard for hex in `printf`-like format specifiers is `%x` and `%X`. What's the motivation for `%h/H`?

Reply:
Was this resolved out-of-band? I'm curious what the rationale to keep %h as.